cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/AuthType.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package io.cdap.cdap.proto.sourcecontrol;
+package io.cdap.cdap.sourcecontrol;
```

Comment:
why do we need to move it out of proto?

Reply:
So it was a combined result of two things:
1. We have a new source control module merged in from https://github.com/cdapio/cdap/pull/14886 and wanted to group the relevant classes in its own place
2. It brings circular dependency issue when I have `RepositoryConfigValidationException` (originally in cdap-proto)extend  `BadRequestException`(in cdap-common). The context: https://github.com/cdapio/cdap/pull/14886#discussion_r1097841578. So when we moved around `RepositoryConfigValidationException`, other classes like `RepositoryConfig`, `RepositoryMeta` that throws the exception also needs to be moved out from cdap-proto. Then I just decided to more them all into the scm module

Reply:
RepositoryConfigValidationException doesn't need to extend BadRequestException. It always needs to be translated into its own response body anyway, meaning it should be caught and handled in the handler

Reply:
sounds good, reverted the changes here